src/comps/ConnectWallet.tsx

```diff
@@ -49,6 +50,25 @@ const WALLET_PROVIDERS = [
   },
 ];
 
+const isMainnetWallet = (addresses: Awaited<ReturnType<getAddresses>>) => {
+  const isBitcoinMainnetAddress =
+    addresses.payment?.address?.startsWith("bc1") ||
+    addresses.payment?.address?.startsWith("3");
```

Comment:
qq: why don't we support P2PKH addresses?

Reply:
Good idea let's incorporate it!